ols/app/models/models.py

```diff
@@ -490,3 +490,49 @@ class SummarizerResponse:
     response: str
     rag_chunks: list[RagChunk]
     history_truncated: bool
+
+
+class CacheEntry(BaseModel):
```

Comment:
it's ok, just the name is a bit misleading, as the real cache entry also contains compound keys etc.

Reply:
Any better suggestion? :)

Reply:
I assume no better name for the moment :)